.circleci/config.yml

```diff
@@ -615,6 +615,9 @@ workflows:
             - be-deps
           e: << matrix.version >>
           driver: mongo
+          extra-env: >-
```

Comment:
The test user and password used by our Mongo tests got hardcoded to the values we use for our CI images at some point. When developing against MongoDB locally I was using the image from the dev scripts (https://github.com/metabase/dev-scripts/blob/master/run-mongo.sh) and had to spend a while trying to figure out why it had stopped working where it had worked in the past. I eventually realized that it was trying to connect to my local image, which does not use a user or password, with the hardcoded user and password.

The right pattern is to make the test user/password/etc. configurable by `MB_<driver>_TEST_` env vars, and then specify them for images that need them

Reply:
Apparently we have a ton of MongoDB tests all over the place -- SSH tests are in CircleCI while we have non-SSH tests in GH actions. Maybe we can consolidate these in the near future and use a matrix or something so we don't have so many duplicate config lines